README.md

```diff
@@ -1,41 +1,44 @@
-# risc0-rust-starter
+# RISC Zero Rust Starter
 
-This is a small example designed to be a minimum starting project for the RISC Zero zero knowledge virtual machine.
-In this example, we use zero knowledge proof techniques to prove that we know the factors of some composite number `N`, without revealing what the factors are.
+The `risc0-rust-starter` repo is a minimal starting project for zero-knowledge software development. In this RISC Zero "Hello World" project, we use zero-knowledge proof techniques to prove that we know the factors of some composite number `N`, without revealing what the factors are. 
 
-For the main project, see [here](https://github.com/risc0/risc0)
+For more information, check out the [main risc0 repo](www.github.com/risc0/risc0) and the [RISC Zero website](http://www.RISCZero.com).
 
-# Building & running
+# Quick Start
 
-Make sure you are using the nightly version of rust:
+Make sure you are using a [nightly](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) version of [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html):
 
 ```
 rustup toolchain install nightly
 rustup override set nightly
 ```
+To build all methods and execute the method within the zkVM, run either of the following options: 
 
-Run
-
+### Option A: Cargo Commands
 ```
-cargo run --bin risc0-build-methods && cargo run --release --bin starter
+$ cargo run --bin risc0-build-methods && cargo run --release --bin starter
```

Comment:
I think we should be consistent about the `$` prompt prefix.

I'm on the fence for this. On one hand it makes it easier to see this is a bash command. On the other, it makes it harder for people to copy/paste the line.

Reply:
Deferring to you on that consistency/alignment decision.

Reply:
OK, for now let's drop the `$` because other places in this same doc don't have it.

Reply:
fixed :)